### PR TITLE
SLOBS optimizations

### PR DESCRIPTION
**Pull request by holiber**

### CODE REVIEW

File: app/services/scene-collections/scene-collections.ts [hunk 1]
```diff
@@ -668,16 +670,12 @@ export class SceneCollectionsService extends Service implements ISceneCollection
     if (this.autoSavePromise) await this.autoSavePromise;
   }
 
-  private async setActiveCollection(id: string) {
+  private async setActiveCollection(id: string, informServer = true) {
```
Comment (andycreeth) [inline app/services/scene-collections/scene-collections.ts:673]: What is `informServer doing?  I don't see it being used.

File: app/services/scene-collections/scene-collections.ts [hunk 2]
```diff
@@ -668,16 +670,12 @@ export class SceneCollectionsService extends Service implements ISceneCollection
...
     const collection = this.collections.find(coll => coll.id === id);
 
     if (collection) {
       if (collection.serverId && this.userService.isLoggedIn) {
-        try {
-          await this.serverApi.makeSceneCollectionActive(collection.serverId);
-        } catch (e) {
-          console.warn('Failed setting active collection');
-        }
```
Comment (andycreeth) [inline app/services/scene-collections/scene-collections.ts:680]: Is there a reason for removing the `try/catch` here?

File: electron-builder/base.config.js [hunk 3]
```diff
@@ -37,7 +38,9 @@ const base = {
   },
   extraMetadata: {
     env: 'production'
-  }
+  },
+  afterPack: 'node_modules/streamlabs-obs-launcher/after_pack.js',
+  afterSign: 'node_modules/streamlabs-obs-launcher/after_sign.js'
```
Comment (andycreeth) [inline electron-builder/base.config.js:43]: Changes to this file should be removed from this PR.

File: updater/bootstrap.ts [hunk 4]
```diff
@@ -182,53 +177,39 @@ async function fetchUpdater(info: IUpdateInfo): Promise<string | null> {
...
-  if (response.statusCode !== 200) {
-    console.log(`Failed to fetch version information - ${response.statusCode}`);
+  const response = await fetch(`${info.baseUrl}/${info.versionFileName}`);
 
+  if (response.status !== 200) {
+    console.log(`Failed to fetch version information - ${response.status}`);
     return null;
   }
 
-  return response.body;
```
Comment (andycreeth) [inline updater/bootstrap.ts:231]: Can you extract all changes to `bootstrap.ts` into a separate PR?  This is the riskiest file in the entire app to change as if it's broken we cannot push new updates.  Any modifications to this file require extremely extensive testing.  I want to do that testing - but I'm not sure we have time to give it the attention it deserves for the next release.

File: app/services/scene-collections/scene-collections.ts (hunk 1 above)
Comment (holiber) [reply app/services/scene-collections/scene-collections.ts:673]: Forgot to delete it from some experiments

File: app/services/scene-collections/scene-collections.ts (hunk 2 above)
Comment (holiber) [reply app/services/scene-collections/scene-collections.ts:680]: After removing "await" keyword it doesn't make sense. I'll add ".catch()" instead

Comment (andycreeth) [reply app/services/scene-collections/scene-collections.ts:680]: Oh of course.  Wasn't reading the change correctly.  ty.

**Pull request merged**